### ISSUE TITLE
Update Directions link behavior on VAMC pages

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#30917]
## Issue Description
For accessibility compliance and usability, when a user clicks on a Directions link, Google maps should be opened in the same tab, rather than in a new one. 

---
## Tasks
- [ ] Update behavior when Veteran clicks on the Directions link

## Acceptance Criteria
- [ ] When a Veteran clicks on the Directions link from a VAMC page, Google Maps will open in the same tab.
- [ ] When a Veteran clicks on the back button from Google maps, users should be returned to the page from which the link was clicked.

**Comment by mmiddaugh** [comment]
This impacts Locations pages also.
